### PR TITLE
Authentication profil page #83

### PR DESCRIPTION
**Pull request by mathben**
- add ssl certificat
- redirect http to https
- fix character page when user connect
- improve third-party auth and user subscribe
and more.

### CODE REVIEW

File: src/web/py_class/db.py [hunk 1]
```diff
@@ -38,18 +51,27 @@ def create_user(self, name, email=None, password_name=None, password_mail=None,
...
-            secure_pass_mail = None
-
-        data = {"email": email, "name": name, "password_name": secure_pass_name, "password_mail": secure_pass_mail,
-                "user_id": user_id, "google_id": google_id, "facebook_id": facebook_id, "twitter_id": twitter_id,
-                "permission": permission}
+        secure_pass = self.generate_password(password) if password else None
+
+        # TODO let user create it and not automatic create
+        empty_character = [{
+            "habilites": [
```
Comment (mathben) [inline src/web/py_class/db.py]: Need to test to add empty character without this defaut information. The creation of character will create the default value.

File: src/web/resources/js/tl_module/profile_ctrl/profile_ctrl.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      };
+      // send command to server
+      $http({
+        method: "post",
+        url: "/cmd/profile/add_new_password",
+        headers: {"Content-Type": "application/json; charset=UTF-8"},
+        data: data,
+        timeout: 5000
+      }).then(function (response/*, status, headers, config*/) {
+        console.info(response.data);
```
Comment (mathben) [inline src/web/resources/js/tl_module/profile_ctrl/profile_ctrl.js]: Commenter la console.info

File: src/web/py_class/db.py (hunk 1 above)
Comment (mathben) [reply src/web/py_class/db.py]: Done

File: src/web/resources/js/tl_module/profile_ctrl/profile_ctrl.js (hunk 2 above)
Comment (mathben) [reply src/web/resources/js/tl_module/profile_ctrl/profile_ctrl.js]: Ignore, usefull for debug.

**Comment by mathben** [comment]
C'est fixé pour l'émail et l'affichage des erreurs.
